charts/redis/templates/logger-redis-secret-creds.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     "helm.sh/hook": pre-install
 data: {{ if eq .Values.global.logger_redis_location "on-cluster"}}
   password: {{ randAlphaNum 32 | b64enc }}{{ else if eq .Values.global.logger_redis_location "off-cluster"}}
-  db: {{ .Values.db | b64enc }}
-  password: {{ .Values.password | b64enc }}
-  host: {{ .Values.host | b64enc }}
-  port: {{ .Values.port | b64enc }}{{ end }}
+  db: {{ .Values.logger.redis.db | b64enc }}
```

Comment:
agree with @kmala 
I did the same for `fluentd` syslog endpoint https://github.com/deis/workflow/pull/631/commits/67b658f692c65e274b486e7f3e91705bd1204921#diff-d3168eb1e6c9201c7558cc356cd625daR135